### PR TITLE
Add test coverage of yii\helpers\BaseArrayHelper

### PR DESCRIPTION
**Pull request by ilyar**
| Q             | A
| ------------- | ---
| Is bugfix?    | no
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Test coverage?   | up 
| Fixed issues  |

### CODE REVIEW

File: tests/framework/helpers/ArrayHelperTest.php [hunk 1]
```diff
@@ -51,6 +51,9 @@ protected function setUp()
 
     public function testToArray()
     {
+        $dataArrayable = $this->getMock('yii\\helpers\\ArrayHelper');
+        $this->assertEquals([], ArrayHelper::toArray($dataArrayable));
```
Comment (cebe) [inline tests/framework/helpers/ArrayHelperTest.php]: what are you testing here? converting empty object to an empty array? if so, `new stdClass` would be more intuitive than calling getMock() for it.

**Review by cebe — CHANGES_REQUESTED** [review]

File: tests/framework/helpers/ArrayHelperTest.php (hunk 1 above)
Comment (ilyar) [inline tests/framework/helpers/ArrayHelperTest.php]: This is test line https://github.com/yiisoft/yii2/blob/master/framework/helpers/BaseArrayHelper.php#L87 I made a mistake in the name of the class should be interface `yii\base\Arrayable`.

Comment (ilyar) [inline tests/framework/helpers/ArrayHelperTest.php]: done

**Review by cebe — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**